src/cms/templates/pages/page_tree.html

```diff
@@ -47,16 +52,19 @@ <h2 class="heading font-normal mb-2">{% trans 'Pages' %}</h2>
                         {% trans 'Create page' %}
                     </button>
                 {% endif %}
-            </div>
-        {% endif %}
+            {% endif %}
+        </div>
     </div>
 </div>
+<div id="filter-form-container" class="{% if not filter_form.has_changed %}hidden{% endif %} w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white">
+    {% include 'pages/_page_filter_form.html' %}
+</div>
 
 <div class="table-listing">
     <table class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white table-auto">
         <thead>
             <tr class="border-b border-solid border-gray-200">
-                <th class="text-sm text-left uppercase py-3 pl-4 pr-2 min"></th>
+                {% if enable_drag_and_drop %}<th class="text-sm text-left uppercase py-3 pl-4 pr-2 min"></th>{% endif %}
```

Comment:
```suggestion
                <th class="text-sm text-left uppercase py-3 pl-4 pr-2 min"></th>
```
If you apply my previous suggestion, this table column is required.